### ISSUE TITLE
Use Twitter bootstrap instead of blueprint

### ISSUE THREAD

**Issue by bevenky** [sehmaschine/django-grappelli#30]
For the grids and styles, would be great to use twitter bootstrap instead of blueprint as it will come out with more themes in near future and is more actively updated.

**Comment by nym** [comment]
Please switch to Bootstrap!

**Comment by parsch** [comment]
@nym sorry, this issue has been closed for several reasons, mentioning two main important ones:
1) the boostrap-issue has been discussed a lot, we always expressed our concerns about possibilities to base grappelli on bootstrap (reminder: you'll be able to implement/transfer 80% right away, but you'll fail when it comes to the last 20% and grappelli is about the last 20%). no one came up with a consistent solution that far, but feel free to surprise us ...
2) the future of grappelli has to be something different than bootstrap, as bootstrap is quite outdated when it comes to modern frontend frameworks/technologies.

**Comment by nym** [comment]
I no longer use grappelli, so I have no skin in the game, but given that it's about a year and a half since I posted this, I wanted to bring up the point that bootstrap is still alive and kicking. 

ThemeForest has 234 Admin themes that are compatible with Bootstrap:

https://themeforest.net/category/site-templates/admin-templates?utf8=%E2%9C%93&referrer=search&view=grid&compatible_with%5B%5D=Bootstrap

Stack Overflow has the question "What custom themes are there for django admin"?

The lack of ability to theme Grappelli was a big PITA at the time I was requesting the feature. There are no real options out there, other than rolling your own. If there is another option out there with more available themes for sale and for free, I'd love to hear about it. I tried to google alternatives, and they all appeared to be fairly niche (lacking available themes in particular).
